### PR TITLE
 Focus Index Update: Redirect, Radio Button Control, 'Continue' Button

### PR DESCRIPTION
**Pull request by fouziiah**
### Changes

- Updated focus index page to redirect to the student path.

- Added stimulus controller for enabling/disabling radio buttons.

- Changed the f.submit button name to 'Continue'.

### Link to Issue 
https://app.asana.com/0/1206510424537042/1207247077374516/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 

**Button Disabled: No Focus Selected**
Screenshot 2024-05-22 at 1 39 41 PM

**Focus Selected: Button Enabled**
Screenshot 2024-05-22 at 1 41 58 PM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/focuses/index.html.erb [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
+          <input type="radio" id="<%= "radio_#{focus.id}"%>" name="student_focus[focus_id]" value="<%= focus.id %>" class="mx-2 w-2 h-4 border border-solid border-gray-200 bg-black" data-focus-form-target="radio" data-action="focus-form#change">
           <label for="<%= "radio_#{focus.id}" %>" ><%= focus.name %></label>
         </li>
       <% end %>
     </ul>
-    <div class= "absolute bottom-0 right-0 mb-7 mr-7">
-    <%= f.submit "Save", class: "text-white bg-green-500 px-4 py-2 rounded-lg " %>
-    </div>
+    <div class="absolute bottom-0 right-0 mb-7 mr-7">
+    <%= f.submit "Continue", class: "text-white bg-green-500 px-4 py-2 rounded-lg opacity-25", disabled: true, data: { focus_form_target: "submitButton" }  %>
```
Comment (edithemmings) [inline app/views/focuses/index.html.erb:27]: The ticket says the button should be disabled "if the student does not have any student focuses". Looks like it is always disabled by default here

File: app/controllers/focuses_controller.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def build_and_save_student_focus
     @student_focus = @student.student_focuses.build(student_focus_params.merge(active: true,
                                                                                educator_id: current_user.id))
     if @student_focus.save
-      redirect_to student_focuses_path(@student), flash: { success: 'Focus was successfully updated.' }
+      redirect_to student_path, flash: { success: 'Focus was successfully updated.' }
```
Comment (edithemmings) [inline app/controllers/focuses_controller.rb:35]: This is erroring out for me locally. Did you mean to pluralize it?

Comment (fouziiah) [reply app/controllers/focuses_controller.rb:35]: fixed it: missing 's' to "students"

File: app/views/focuses/index.html.erb (hunk 1 above)
Comment (fouziiah) [reply app/views/focuses/index.html.erb:27]: Got it, will correct that

**Review by crestu — DISMISSED** [review]
tested it out and everything works well! continue button not working unless a focus is selected and is saving the focus and redirecting to the students page! nice work!!

**Comment by edithemmings** [comment]
Merge conflicts. Going to wait to review until those are resovled

**Review by edithemmings — APPROVED** [review]
Nice!!! Works like a charm!

**Review by crestu — APPROVED** [review]
nice work!

**Pull request merged**